storage/bucket.go

```diff
@@ -782,6 +786,15 @@ type BucketWebsite struct {
 	NotFoundPage string
 }
 
+// BucketCustomPlacementConfig holds the bucket's custom placement
+// configuration for Custom Dual Regions. See
+// https://cloud.google.com/storage/docs/locations for more information.
+type BucketCustomPlacementConfig struct {
```

Comment:
Perhaps just `CustomPlacementConfig` for brevity (and I think that's what it's called in the API itself, the `Bucket` is just from the generated code).

Reply:
sounds good, updated!